cirrus/server/tests/conftest.py

```diff
@@ -1,7 +1,7 @@
 from fastapi.testclient import TestClient
 from pytest import fixture
 
-from cirrus.main import app
+from ..cirrus.main import app
```

Comment:
Why `..`?

Reply:
added some `__init__.py` to fix module issue